lib/scrabble.rb

```diff
@@ -0,0 +1,87 @@
+module Scrabble
+  class Scoring
+    attr_reader :values
+
+    VALUES = {
+      1 => ["A", "E", "I", "O", "L", "N", "R", "S", "T"],
+      2 => ["D", "G"],
+      3 => ["B", "C", "M", "P"],
+      4 => ["F", "H", "V", "W", "Y"],
+      5 => ["K"],
+      8 => ["J", "X"],
+      10 => ["Q", "Z"]
+    }
+
+    def self.score(input_word)
+      if !(input_word.to_s.match(/[a-zA-Z]/))
+        raise ArgumentError.new "Invalid character in word"
+      end
+
+      if input_word.length == 7
+        score = 50
+      else
+        score = 0
+      end
+
+      input_letters = input_word.split("")
+      input_letters.each do |letter|
+
+        if VALUES[1].include? letter.upcase
+          score += 1
+        elsif VALUES[2].include? letter.upcase
+          score += 2
+        elsif VALUES[3].include? letter.upcase
+          score += 3
+        elsif VALUES[4].include? letter.upcase
+          score += 4
+        elsif VALUES[5].include? letter.upcase
+          score += 5
+        elsif VALUES[8].include? letter.upcase
+          score += 8
+        elsif VALUES[10].include? letter.upcase
+          score += 10
+        else
+          raise ArgumentError.new "Invalid letter in word"
+        end
```

Comment:
What if values changed and you updated the `VALUES` data to include ` 9 => ["XXX"]`? Then you would need to update this logic in order to have a new line `elsif VALUES[9].include? ...` etc. There's opportunity here to refactor this so that you don't need to have all of these conditionals.